frontend/src/components/atoms/Checkbox/Checkbox.stories.js

```diff
@@ -21,28 +21,44 @@ const Template = (args) => (
 
 export const Unchecked = Template.bind({});
 Unchecked.args = {
-  labelText: 'Unchecked',
+  label: 'Unchecked',
+  name: 'Unchecked',
   checked: false,
   disabled: false,
+  mt: 0,
```

Comment:
Marginy wszędzie do wywalenia. Niech to kontrolują style.

Reply:
wywalone